django_project/settings.py

```diff
@@ -12,22 +12,34 @@
 
 from pathlib import Path
 from environ import Env
-
+import os
+from django.core.exceptions import ImproperlyConfigured
 from django.utils.translation import ugettext_lazy as _
 
+
+def require_env(name):
+    """Raise an error if the environment variable isn't defined"""
+    value = os.getenv(name)
```

Comment:
why `os.getenv()` when we used `env()` before?

Reply:
Not sure how to explain, env() used the Env package and now we went through this solution: https://stackoverflow.com/questions/37473108/how-do-you-add-environment-variables-to-your-django-project/37473364#37473364

Reply:
This is just an example I sent you to create validation function. You are not bound to use this solution character by character. Of course you should use `env` to not break any functionality and just move it to this function.

Reply:
Got it, pushed a fix.